Update to reflect new categories

andyvan-ph opened the pull request:
## Changes

Adding new categories information to the blog guide

## Checklist
- [ ] Titles are in [sentence case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case)
- [ ] Feature names are in [title case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case)
- [ ] Words are spelled using American English
- [ ] I have checked out our [style guide](https://github.com/PostHog/posthog.com/blob/master/STYLEGUIDE.md)

corywatilo reviewed (changes requested):
Since we use [sentence case](https://posthog.com/handbook/company/communication#writing-style#writing-style) (I already updated the casing for the categories on the actual blog last week), we should match them here!

andyvan-ph commented:
> Since we use [sentence case](https://posthog.com/handbook/company/communication#writing-style#writing-style) (I already updated the casing for the categories on the actual blog last week), we should match them here!

Ah yes, I noticed. Well spotted and thank you. I've updated this handbook entry now.
